src/hooks/useAffiliatesInfo.ts

```diff
@@ -60,12 +62,87 @@ export const useAffiliatesInfo = (dydxAddress?: string) => {
     }
   };
 
+  const fetchProgramStats = async () => {
+    const endpoint = `${affiliatesBaseUrl}/v1/community/program-stats`;
+
+    try {
+      const res = await fetch(endpoint, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'GET',
+      });
+
+      const data = await res.json();
+      return data;
+    } catch (error) {
+      log('useAffiliatesInfo/fetchProgramStats', error, { endpoint });
+      throw error;
+    }
+  };
+
+  const fetchAccountStats = async () => {
+    const endpoint = `${affiliatesBaseUrl}/v1/leaderboard/account/${dydxAddress}`;
+
+    try {
+      const res = await fetch(endpoint, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'GET',
+      });
+
+      const data = await res.json();
+
+      return data;
+    } catch (error) {
+      log('useAffiliatesInfo/fetchAccountStats', error, { endpoint });
+      throw error;
+    }
+  };
+
+  const fetchLastUpdated = async () => {
+    const endpoint = `${affiliatesBaseUrl}/v1/last-updated`;
+
+    try {
+      const res = await fetch(endpoint, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'GET',
+      });
+
+      const data = await res.json();
+
+      return data;
+    } catch (error) {
+      log('useAffiliatesInfo/fetchLastUpdated', error, { endpoint });
+      throw error;
+    }
+  };
+
   const affiliateMetadataQuery = useQuery({
-    queryKey: ['affiliatesMetadata', dydxAddress],
+    queryKey: ['affiliateMetadata', dydxAddress, compositeClient],
     queryFn: fetchAffiliateMetadata,
     enabled: Boolean(compositeClient && dydxAddress),
   });
 
+  const programStatsQuery = useQuery({
+    queryKey: ['programStats'],
+    queryFn: fetchProgramStats,
+  });
+
+  const affiliateStatsQuery = useQuery({
+    queryKey: ['accountStats', dydxAddress],
+    queryFn: fetchAccountStats,
+    enabled: Boolean(dydxAddress),
+  });
+
+  const lastUpdatedQuery = useQuery({
+    queryKey: ['lastUpdated'],
+    queryFn: fetchLastUpdated,
+  });
```

Comment:
should we add `affiliatesBaseUrl` to the `queryKeys` and `enabled` params?

Reply:
i think its fine to not include it since we dont expect the base URL to change dynamically, and generally keeping query keys shorter is better